### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by fherrerao**
# Hi There
The project has the next features:

- Use of Bootstrap for styling.
- Build Home and About pages.
- The home page should have a link in the menu to the about page.
- The logo in the header links to the home page.
- The section "Featured speakers" was created dynamically in JavaScript.

## Vide Description

Check the video description of the project for general information about it:

https://www.loom.com/share/728eec5d4fc24fa9aa60e960ae6725e6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +34,6 @@ Give a ⭐️ if you like this project!
 
 - Hat tip to anyone whose code was used
 - Inspiration
```
Comment (aliabbani) [inline README.md]: The Acknowledgements section should give appropriate credit to[ Cindy Shin](https://www.behance.net/adagio07) the  [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Please make sure to add credit to the original designer.

File: README.md [hunk 2]
```diff
@@ -1,66 +1,30 @@
...
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Html
+- Css
+- Javascript
+- -Bootstrap
```
Comment (aliabbani) [inline README.md]: Your readme file looks good👍. Please make it perfect by removing the ( - ) before the Bootstrap word.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+        </section>
+
+        <footer class="container gap-5 d-lg-flex my-2 justify-content-end">
+            <a class="navbar-brand text-secondary mx-5" href="#"><i class="fas fa-tram fa-3x">travel</i></a>
+            <div class="mt-3 mx-5">
+                <p class="fw-bold text-center">2022 Creative Commons Korea. Some Rigths Reserved. <span></span> </p>
+                <p class="text-center">Lorem ipsum dolor sit amet consectetur adipisicing elit. In quis quos ipsam,
+                    maxime voluptatum.</p>
+            </div>
+        </footer>
```
Comment (aliabbani) [inline index.html]: The footer section in the mobile version should look almost identical to the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

Your design:
image

The original design : 
image

Please make sure to follow the original design as you can just make it horizontally.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+                <div class="vector"></div>
+            </div>
+            <ul class="d-lg-flex justify-content-center align-items-center gap-5">
+                <li class="li-companies text-center mt-4"><img src="src/img/mozilla.png" alt=""></li>
+                <li class="li-companies text-center mt-4"><img src="src/img/google.png" alt=""></li>
+                <li class="li-companies text-center mt-4"><img src="src/img/naver.png" alt=""></li>
+                <li class="li-companies text-center mt-4"><img src="src/img/daumkakao.png" alt=""></li>
+                <li class="li-companies text-center mt-4"><img src="src/img/airbnb.png" alt=""></li>
+            </ul>
+        </section>
```
Comment (aliabbani) [inline index.html]: The partner section in the mobile version should look almost identical to the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

Your design:
image

The original design : 
image

Please make sure to follow the original design as you can just make it horizontally.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="bg-image">
+            <div class="container pt-5">
+                <h1 class="title">"Hello! Sharing world"</h1>
+                <h2 class="sub-title">CREATIVE COMMONS GLOBAL SUMMIT 2022</h2>
```
Comment (aliabbani) [inline index.html:81]: The pages should look almost identical to the original design. Please make sure to add text effect for the h2 as with the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

File: index.html [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+                            <a class="nav-link" href="#">Sponsor</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">News</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="btn btn-outline-danger border-4">CC Campaign</a>
+                        </li>
+                    </ul>
+                </div>
```
Comment (aliabbani) [inline index.html:73]: The mobile menu should have a close (X) button that closes the menu. Please add the close (X) button with its functionality.

File: about.html [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+        </section>
+
+    </main>
+    <footer class="gap-5 d-lg-flex justify-content-end bg-dark py-3">
+        <a class="navbar-brand text-secondary mx-5" href="#"><i class="fas fa-tram fa-3x">travel</i></a>
+        <div class="mt-3 mx-5">
+            <p class="fw-bold text-white text-center">2022 Creative Commons Korea. Some Rigths Reserved. <span></span> </p>
+            <p class="text-white text-center" >Lorem ipsum dolor sit amet consectetur adipisicing elit. In quis quos ipsam, maxime voluptatum.</p>
+        </div>
+    </footer>
```
Comment (aliabbani) [inline about.html]: The footer design of the about page should look identical to the footer design of the home page **except** for the colors. Please make sure the padding and margin are the same.

**Review by aliabbani — CHANGES_REQUESTED** [review]
Hi @fherrerao, 

Good job so far! I can see real talent here. You really did a great job 🥇 .
However, there are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there! 

 ## Required changes 
_Check the comments under the review._ 

Cheers and Happy coding!👏👏👏 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rustam-tolipov — APPROVED** [review]
## Hi @fherrerao  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by fherrerao** [comment]
# Hi @Rustamxon7 
Thanks for your excellent job! 💯

**Pull request merged**